src/CommandingOfficers/AWBW/OS/Andy.java

```diff
@@ -0,0 +1,117 @@
+package CommandingOfficers.AWBW.OS;
+
+import java.util.ArrayList;
+
+import CommandingOfficers.*;
+import CommandingOfficers.CommanderAbility.CostBasis;
+import CommandingOfficers.AWBW.AWBWCommander;
+import Engine.GameScenario;
+import Engine.UnitMods.UnitDamageModifier;
+import Engine.UnitMods.UnitModifier;
+import Engine.UnitMods.UnitMovementModifier;
+import UI.UIUtils;
+import Terrain.MapMaster;
+import Units.Unit;
+
+public class Andy extends AWBWCommander
```

Comment:
Some part of my brain is _bothered_ that we have an AWBW Andy. It's not even an AW game. Why is it different? Are all the other COs different in some subtle way such that we need a third (fourth?) nearly-indistinguishable variant of each of them? Is anyone going to even notice the differences in gameplay? Is this what happens when I review code after midnight? Or just when I review your design decisions specifically?

Reply:
Forgive me being laconic, as Github helpfully deleted my comment when I deleted the extra copy.

It's different because AWBW was implemented before a lot of the various specific mechanics were well-known, then inertia.

There are various stat differences (Sturm, Drake, Flak)
The COP cost bug doesn't exist (benefiting Adder a lot)
Universal +10/10 like AW3, except it's taken out of universal attack boosts for some reason.
Charge isn't like AW3, as well, so you can't just use that one.

I can't really speak to whether most will notice, but someone actually implemented no healing round-up to have a more accurate playtest.

Reply:
So uh, why did Andy1 miss the boat?

Reply:
Because AW1 is special (i.e. will require more changes), and I am trying to keep PRs not too gigantic.
These were mostly already supported by the engine.